.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Report a bug or an issue you've found with Trino Python client
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is. What script did you run? What happened?
+
+### Steps To Reproduce
+In as much detail as possible, please provide steps to reproduce the issue. Sample code that triggers the issue, relevant server settings, etc is all very helpful here.
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
```

Comment:
I think this should go first, instead of `describe the bug`. After the expected behavior, users should describe the actual behavior. If the bug is not an actual issue, that should make it clear.

Reply:
Good point